client/src/js/app.js

```diff
@@ -5,7 +5,8 @@ var bhima = angular.module('bhima', [
   'tmh.dynamicLocale', 'ngFileUpload', 'ui.grid',
   'ui.grid.selection', 'ui.grid.autoResize', 'ui.grid.resizeColumns',
   'ui.grid.edit', 'ui.grid.grouping', 'ui.grid.treeView', 'ui.grid.cellNav',
-  'ui.grid.pagination', 'ui.grid.moveColumns', 'angularMoment', 'ngMessages',
+  'ui.grid.pagination', 'ui.grid.moveColumns', 'ui.grid.exporter',
```

Comment:
Your analysis is correct - for the trial balance, it is more work to export a CSV from the server side.

My biggest worry with this change is that, as a developer, I do not know when to use the server-side CSV rather than the client-side CSV.  It creates another question to ask during development and something else to test....